Developer-Guide.md

```diff
@@ -301,6 +310,173 @@ $ sudo systemctl restart docker
 $ sudo docker run -ti --runtime kata-runtime busybox sh
 ```
 
+# If you want to run Kata Containers with Kubernetes
+
+Now that Kata Containers has been installed on your system, you need some
+extra components on top to make this work with Kubernetes.
+
+## Install CNI
+
+Kubernetes networking relies on CNI plugins, meaning you have to install
+them and choose which one will fit your needs. Follow the documentation
+[here](https://github.com/containernetworking/cni#running-the-plugins) and
+you can also refer to our CI script
+[here](https://github.com/kata-containers/tests/blob/master/.ci/install_cni_plugins.sh)
+if you are interested in understanding the full setup.
+
+## Install a CRI implementation
+
+Kata Containers runtime is an OCI compatible runtime and cannot directly
+interact with the CRI API level. For this reason, we need to rely on a
+CRI implementation translating CRI into OCI. There are two supported ways
+called [CRI-O](https://github.com/kubernetes-incubator/cri-o) and
+[CRI-containerd](https://github.com/containerd/cri). It is up to you to
+choose the one that you want, but you have to pick one, and make appropriate
+changes to make sure they will rely on Kata Containers runtime.
+
+### CRI-O
+
+If you picked CRI-O, please follow the instructions
+[here](https://github.com/kubernetes-incubator/cri-o/blob/master/tutorial.md)
+to get it properly installed.
+
+Now, the idea is that you will have to modify CRI-O configuration with
```

Comment:
s/Now, the idea is.../Now you need to modify/

Reply:
Lines 352-356 suggested rewrite (hopefully I did not change meaning here): 

Once you have installed CRI-O, you need to modify the CRI-O configuration with information about different container runtimes. By default, we choose `runc`, but in this case we also specify Kata Containers runtime to run _untrusted_ workloads. In other words, this defines an alternative runtime to be used when the

Reply:
done